languages/python/exercises/concept/currency-exchange/exchange_test.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+from exchange import *
+
+
+class TestExchange(unittest.TestCase):
+    
+    # Problem 1
+    def test_estimate_value(self):
+        input_data = [
+            # input : budget, exchange_rate
+            (100000, 0.84),
+            (700000, 10.1)
+        ]
+        output_data = [119047, 69306]
+        for input, output in zip(input_data, output_data):
+            with self.subTest(input=input, output=output):
+                self.assertEqual(int(estimate_value(input[0], input[1])), output)
+
+    # Problem 2
+    def test_get_change(self):
+        input_data = [
+            # input : budget, exchanging_value
+            (463000, 5000)
+        ]
+        output_data = [458000]
+        for input, output in zip(input_data, output_data):
+            with self.subTest(input=input, output=output):
+                self.assertEqual(get_change(input[0], input[1]), output)
+
+    # Problem 3
+    def test_get_value(self):
+        input_data = [
+            # input : denomination_value, number_of_bills
+            (10000, 128),
+        ]
+        output_data = [1280000]
+        for input, output in zip(input_data, output_data):
+            with self.subTest(input=input, output=output):
+                self.assertEqual(get_value(input[0], input[1]), output)
```

Comment:
It might be nice to add some more input/output cases here for these two test cases.  Otherwise, you should eliminate the `subtest` loops for each, since there is only one set of inputs to each test.